account_fiscal_year_closing/views/account_fiscalyear_closing_template_views.xml

```diff
@@ -17,7 +17,7 @@
     <record id="view_account_fiscalyear_closing_template_form" model="ir.ui.view">
         <field name="model">account.fiscalyear.closing.template</field>
         <field name="arch" type="xml">
-            <form string="Fiscal year closing">
+            <form>
```

Comment:
Motivo di questa modifica?

Reply:
> Meaning of this change?

https://github.com/OCA/account-closing/pull/276

Reply:
This is a form, perhaps was intended to fix this row? 
account_fiscal_year_closing/views/account_fiscalyear_closing_views.xml:10 Deprecated "<tree string=..." - [xml-deprecated-tree-attribute]

Reply:
> This is a form, perhaps was intended to fix this row? account_fiscal_year_closing/views/account_fiscalyear_closing_views.xml:10 Deprecated "<tree string=..." - [xml-deprecated-tree-attribute]

sorry wrong fix, now should be ok

Reply:
Thanks